### ISSUE TITLE
Failed to delete character when mantissaLength = 0

### ISSUE THREAD

**Issue by yosemiteyss** [caseyryan/flutter_multi_formatter#91]
Hi, I want to implement a simple thousands formatter without any decimal place using `CurrencyInputFormatter(mantissaLength: 0)`. But then, character can no longer be removed from text field. Any solution to this?

**Comment by caseyryan** [comment]
Hi! What version of the package do you use? Make sure you're using the latest

**Comment by caseyryan** [comment]
Hi!. Yes, I've just checked this without a trailing or leading symbol. There is a bug

**Comment by yosemiteyss** [comment]
I am able to reproduce it using the latest version.
```
import 'package:flutter/material.dart';
import 'package:flutter_multi_formatter/flutter_multi_formatter.dart';

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      darkTheme: ThemeData.dark(),
      theme: ThemeData.light(),
      home: const MyHomePage(),
      debugShowCheckedModeBanner: false,
    );
  }
}

class MyHomePage extends StatelessWidget {
  const MyHomePage({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: const Text('Flutter app'),
      ),
      body: Center(
        child: TextFormField(
          keyboardType: TextInputType.phone,
          inputFormatters: [CurrencyInputFormatter(mantissaLength: 0)],
        ),
      ),
    );
  }
}

void main() {
  runApp(const MyApp());
}
```

**Comment by caseyryan** [comment]
fixed in version 2.7.0
